### PR TITLE
Worked on building my carousels and displaying properties in card components

### PR DESCRIPTION
**Pull request by SebasCC99**
lots of files, solid code 😉🙏🏻

### CODE REVIEW

File: client/src/related_items/Card.jsx [hunk 1]
```diff
@@ -27,12 +35,32 @@ const Card = (props) => {
...
   }, [productId])
 
+  useEffect(() => {
+    helpers.getProductReview(productId)
+      .then(res => {
+        setRating(res)
+      })
+  }, [productId])
+
+  const handleClick = (e) => {
```
Comment (luna-moon-1216) [inline client/src/related_items/Card.jsx:45]: Where are you using this ?

File: client/src/related_items/RelatedProducts.jsx [hunk 2]
```diff
@@ -23,9 +23,9 @@ const RelatedProducts = (props) => {
   }, [productId])
 
   return (
-    <div id="Related-Products" >
+    <div id="related-products" >
       <h1 id="related-product-header">Related Products</h1>
-      <div className="related-card">
+      <div id="related-cards">
         {relatedProducts.map(productId => (
```
Comment (luna-moon-1216) [inline client/src/related_items/RelatedProducts.jsx:29]: not sure about this, but can you omit props.relatedProducts?

File: client/src/related_items/Card.jsx (hunk 1 above)
Comment (SebasCC99) [reply client/src/related_items/Card.jsx:45]: Im not yet, thats going to be used to handle the click on the star at the top of the card

File: client/src/related_items/RelatedProducts.jsx (hunk 2 above)
Comment (SebasCC99) [reply client/src/related_items/RelatedProducts.jsx:29]: The reason Im able to do it, is that in this case, relatedProducts is state part of my state, Im not getting from props

File: client/src/related_items/helpers.js [hunk 3]
```diff
@@ -15,6 +25,12 @@ const helpers = {
     const url = `/sedna/products/${id}/styles`;
     const styles = await axios.get(url);
     return styles.data
+  },
+  getProductReview: async (id) => {
+    const url = `/sedna/reviews/meta/?product_id=${id}`;
+    const ratings = await axios.get(url);
+    const stars = calculateRatings(ratings.data.ratings);
+    return stars;
```
Comment (luna-moon-1216) [inline client/src/related_items/helpers.js:33]: love these haha, clean codes!

Comment (SebasCC99) [reply client/src/related_items/helpers.js:33]: Thanks!

**Review by luna-moon-1216 — APPROVED** [review]

**Pull request merged**